recipes/hdf4/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib
+fi
+
+export CXXFLAGS="${CFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export CFLAGS="${CFLAGS} -pipe -O2 -fPIC -I${PREFIX}/include"
+
+chmod +x configure
+
+# The --enable-silent-rules is needed because Travis CI dies on the long output from this build.
+./configure \
+    --disable-static \
+    --enable-linux-lfs \
+    --with-ssl \
+    --with-zlib \
+    --with-jpeg \
+    --disable-netcdf \
+    --disable-fortran \
```

Comment:
Guessing these are just bindings so it can be used from Fortran. Yeah, if no one really uses these, this seems completely fine.